index.js

```diff
@@ -6,13 +6,16 @@ const tmp = require('tmp')
 const x = require('throw-if-missing')
 
 module.exports = ({ exe = 'C:/Program Files (x86)/Cisco/Cisco AnyConnect Secure Mobility Client/vpncli.exe',
-                    server = x`server`, username = x`username`, password = x`password` }) => {
+                    server = x`server`, username = x`username`, password = x`password`, acceptCertificate = `acceptCertificate`, group = `group` }) => {
```

Comment:
`acceptCertificate` should be a boolean.

Also there should be no default value for `group`.